header_fix.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import datetime
+import os
+import shutil
+
+lang_dirs = [
+    'ada',
+    'c',
+    'lua',
+    'csharp',
+    'cpp',
+    'elixir',
+    'elm',
+    'golang',
+    'java',
+    'javascript',
+    'ocaml',
+    'objc',
+    'php',
+    'python',
+    'ruby',
+    'rust',
+    #'typescript', Oh you are so special. You make me feel dumb
```

Comment:
did you mean to commit it? i'm not sure i follow what was the intent here

Reply:
i should have left a better comment.
the typescript grammar subdir is organized slightly different from the rest of them, and there is no need to do the header fixing stuff there, and if we need to do it someday, it's very likely to be very different

Reply:
changed the comments